src/misc/programs.cpp

```diff
@@ -530,13 +539,22 @@ void CONFIG::Run(void) {
 			}
 			Section_line* sec = dynamic_cast <Section_line*>
 				(control->GetSection(std::string("autoexec")));
-
-			for(Bitu i = 0; i < pvars.size(); i++) sec->HandleInputline(pvars[i]);
+			if (!sec) {
+				WriteOut(MSG_Get("PROGRAM_CONFIG_SECTION_ERROR"));
+				return;
+			}
+			for(Bitu i = 0; i < pvars.size(); i++) {
```

Comment:
A space wants to squeak in between `for` and `(` :)

Reply:
Got it, and did a general space-addition cleanup of all the `if (`, `for (`, and `while (` statements.  Although the inner-sections might not be perfect; I was getting dizzy after too much of it!